src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+    @Override
+    public List<String> readFromFile(String fileName) {
+        try {
+            List<String> list = Files.readAllLines(Path.of(fileName));
+            list.remove(0);
```

Comment:
Consider moving remove logic to main method. this service doesn't know that it's reading csv file and that first line is header

Reply:
Not fixed. Best place for removing is `parseTransactions` method

Reply:
not fixed